index.html

```diff
@@ -33,6 +33,7 @@
     </script>
     <link rel="stylesheet" href="/index.css" />
     <link rel="stylesheet" href="/library/prism.css" />
+    <script src="https://unpkg.com/shimport"></script>
```

Comment:
Even though there's another `shimport` usage that avoids this, it doesn't want to work with `type="module"`... didn't have bandwidth to dig into that further on shimport's side of things.

For reference, I'm talking about this (from the shimport readme):

```js
function shim(url) {
  try {
    return new Function(`return import('${url}')`).call();
  } catch (tmp) {
    var s = document.createElement('script');
    s.src = 'https://unpkg.com/shimport';
    s.dataset.main = url
    document.head.appendChild(s);
  }
}
```

```js
// Throws:
TypeError: shim(...) is undefined[Learn More] index.js:21:1

    <anonymous> http://localhost:5000/index.js:21 InnerModuleEvaluation self-hosted:4362 evaluation self-hosted:4315 
```

Reply:
Yeh this is exactly what I tried.. didn't think (or want to really 😝) about just adding the script tag.. but it works nicely for now eh!